### DISCUSSION TITLE
M1 Mac minor issues...

### DISCUSSION THREAD

**Discussion by GoldmanOz** [comfyanonymous/ComfyUI#900] [General]
New to ComfyUI, pretty new to terminal so am finding my way slowly...a couple of issues I'm having on my MacBook Pro..

I can't right click on the grid to open up node choices, change titles, colours etc, the normal Control-Click is not functioning, any idea's ??

Please be aware I'm not overly experienced in the thermal so any suggestions you might have will need to be explained thoroughly 😉 

Cheers

**Comment by pontusi** [comment]
I can confirm that this is an issue, but you can right-click by clicking your trackpad with two fingers (assuming you have not changed your trackpad settings), and that works. I'm so used to right-clicking that way that I didn't even notice this until you pointed it out.

**Reply by GoldmanOz** [reply]
Thank you very much....will give that a try....I'm used to command clicking and thats been the problem. 👍🏻
